packages/components/src/InputFile/InputFile.tsx

```diff
@@ -190,30 +197,49 @@ export function InputFile({
   }
 
   async function uploadFile(file: File) {
```

Comment:
Any ideas on how to simplify this function? Or like self-documenting. It's switching between 2 code paths now and it's getting a lil hard to read.

Reply:
You may be able to use the `axios(config)` or `axios.request(config)` api to simplify the branching code paths slightly

Reply:
I think you still get branching logic like that. Because if it's a POST request, we have `formdata` to pass in. If it's a PUT request, we need to pass in `headers` and the file is the payload. We just move the branching logic to how we build out our `config` options for the `axios.request` method. But perhaps that can be moved into a separate function? I'll continue looking into this

Reply:
I removed the branching logic from the `uploadFile` method, and created a new method called `createAxiosConfig` in order to generate the correct input for the `axios.request` method